docs/source/pillars/information_governance.md

```diff
@@ -91,6 +91,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
   - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
   - Recommended
+* - You log data access queries and publish these logs
+  - Transparency is a key principle for an environment that is trusted for research.
+    With these logs publicly available, there is transparency around how the data was used, and any actions that could compromise the security of the sensitive data.
```

Comment:
Again, I'm not sure if we can make a statement this strong from the requirement.

I wouldn't be confident that a list of access events alone could tell you how the data was being used or about security risk.

Reply:
Agreed, there's a lot of faith being put in the use of log files here. Useful logs can be helpful but a *lot* of work has to be put in to make them useful.